src/components/Navbar/Navbar.tsx

```diff
@@ -68,36 +47,34 @@ const Navbar = (props: Props) => {
       </NavDropdown>
     )
   }
-
   return (
     <NavbarRB bg={bg} variant={variant}>
-      <NavbarRB.Brand href={brand.href} onClick={brand.onClick}>
+      <NavbarRB.Brand href={brand.href} onClick={brand.onClick} style={{ cursor: 'pointer' }}>
         {brand.src ? (
           <img
             alt={brand.label}
             src={brand.src}
             width="28"
             height="28"
-            className="d-inline-block align-top"
+            className="d-inline-block align-top mr-3"
           />
         ) : (
           ''
         )}
-        {` ${brand.label}`}
+        <span style={{ color: brand.color }}>{`${brand.label}`}</span>
       </NavbarRB.Brand>
-
       <NavbarRB.Collapse id="responsive-navbar-nav">
         <Nav className="mr-auto">{navLinks.map((link, index) => getNavLinks(link, index))}</Nav>
         <Nav>
           <Form inline>
             <FormControl
```

Comment:
We might want to consider refactoring this text box to be our own TextBox component. Do you want to do that in this pull request. Otherwise, we can open an issue and do it later

Reply:
I was giving this one a shot but I am a little lost on the necessary props and how what will be used as TextBox so I'm not sure if i commit my best attempt with atleast the basic structure in place or if we leave it for another issue. Thanks for the feedback!